app/jobs/europeana/picture_versions_cleanup_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Europeana
+  class PictureVersionsCleanupJob < ActiveJob::Base
```

Comment:
Could we add a note here as a reminder that this is only needed in the code base temporarily, until our live deployments have all had the job executed, after which time this will no longer be needed.